[2024-01-23T18:40:12.207+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data manual__2024-01-23T18:32:06.410806+00:00 [queued]>
[2024-01-23T18:40:12.252+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data manual__2024-01-23T18:32:06.410806+00:00 [queued]>
[2024-01-23T18:40:12.269+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-23T18:40:12.391+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_stackoverflow_data> on 2024-01-23 18:32:06.410806+00:00
[2024-01-23T18:40:12.429+0000] {standard_task_runner.py:60} INFO - Started process 3913 to run task
[2024-01-23T18:40:12.442+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_stackoverflow_data', 'get_stackoverflow_data', 'manual__2024-01-23T18:32:06.410806+00:00', '--job-id', '440', '--raw', '--subdir', 'DAGS_FOLDER/loading_data_to_aws_S3.py', '--cfg-path', '/tmp/tmpm8cuq1bo']
[2024-01-23T18:40:12.455+0000] {standard_task_runner.py:88} INFO - Job 440: Subtask get_stackoverflow_data
[2024-01-23T18:40:12.600+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_stackoverflow_data.get_stackoverflow_data manual__2024-01-23T18:32:06.410806+00:00 [running]> on host 9924936b75b5
[2024-01-23T18:40:12.840+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='charlotte' AIRFLOW_CTX_DAG_ID='fetch_stackoverflow_data' AIRFLOW_CTX_TASK_ID='get_stackoverflow_data' AIRFLOW_CTX_EXECUTION_DATE='2024-01-23T18:32:06.410806+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-23T18:32:06.410806+00:00'
[2024-01-23T18:40:13.474+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-23T18:40:13.493+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=fetch_stackoverflow_data, task_id=get_stackoverflow_data, execution_date=20240123T183206, start_date=20240123T184012, end_date=20240123T184013
[2024-01-23T18:40:13.535+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-23T18:40:13.648+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
